akvo/rsr/migrations/0079_auto_20160620_1418.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def sector_validation(apps, schema_editor):
+    """ Remove sector from RSR validation set """
+    ProjectEditorValidation = apps.get_model('rsr', 'ProjectEditorValidation')
+
+    sector_validators = ['rsr_sector', 'rsr_sector.sector_code', 'rsr_sector.vocabulary']
+
+    for v in sector_validators:
+        ProjectEditorValidation.objects.filter(validation_set_id=1).get(validation__exact=v).delete()
```

Comment:
Something like this might be safer:

```
validation = ProjectEditorValidation.objects.filter(validation_set_id=1, validation__exact=v)
if validation:
    validation.delete()
```